README.md

```diff
@@ -1,24 +1,67 @@
 # Flight Engine
 Mock flight data delivered simply and quickly without a database.
 
+## Let's Get Started
+
+## 👉  [I just want flight data](#deploy-flight-engine-and-use-now)
+
+## 👉  [I want a backend that I can customize](#running-flight-engine-locally)
```

Comment:
Let's make these H3s to avoid the funky line under them

```suggestion
### 👉  [I just want flight data](#deploy-flight-engine-and-use-now)

### 👉  [I want a backend that I can customize](#running-flight-engine-locally)
```